security/permissions/src/main/java/io/quarkus/ts/openshift/security/basic/permissions/resources/ClassAnnotationPermissionAllowedResource.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.ts.openshift.security.basic.permissions.resources;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+import io.quarkus.security.PermissionsAllowed;
+
+@Path("/class-permission-allowed")
+@PermissionsAllowed({ "read:all", "create" })
+public class ClassAnnotationPermissionAllowedResource {
+
+    public static final String RESPONSE_STRING = "Permitted";
```

Comment:
similar comment that I did in `ClassAnnotationPermissionAllowedInclusiveResource` class